### PR TITLE
GitHub Actions: run skeleton tests after normal tests in the same build.

### PR DESCRIPTION
**Pull request by skvadrik**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -64,18 +56,10 @@ jobs:
             build-type: Release
             build-dir: .build
             cxx-flags: ${{ null }}
-            skeleton: ${{ false }}
+            skeleton: ${{ true }}
```
Comment (sergeyklay) [inline .github/workflows/ci.yml:59]: Here and in all other places: could you please rename `linux-gcc-release-ootree` to  `linux-gcc-release-ootree-skeleton` to follow common naming pattern?

Comment (skvadrik) [reply .github/workflows/ci.yml:59]: Done!

**Review by sergeyklay — APPROVED** [review]
I'm not sure about `"${{ matrix.skeleton }}" == true`. However, if I'm wrong and it actually works as you expect - great!

**Comment by sergeyklay** [comment]
Just for your info: There is an ability to cancel builds in case of force-push

image

**Comment by skvadrik** [comment]
> Just for your info: There is an ability to cancel builds in case of force-push

Ah, I was wondering about that. I hoped it would auto-cancel, but apparently not (or maybe they get cancelled, but still show in the UI). :)

**Comment by skvadrik** [comment]
> I'm not sure about `"${{ matrix.skeleton }}" == true`

Seems to work as expected (skeleton build runs skeleton tests, non-skeleton one doesn't). Are you worries about the usual shell things like empty variable on the left hand side, or not quoting `true`? Maybe it should be `"x${{ matrix.skeleton }}" == "xtrue"`.

**Comment by sergeyklay** [comment]
> Maybe it should be `"x${{ matrix.skeleton }}" == "xtrue"`.

Yeah, I usually prefer something like this. But, if everything is OK, then great!

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -275,11 +259,14 @@ jobs:
...
-
-      - name: Skeleton Validation
-        if: startsWith(runner.os, 'Windows') == false && matrix.skeleton == true
+      - name: Test
+        if: startsWith(runner.os, 'Windows') == false
         working-directory: ${{ matrix.build-dir }}
-        run: ./run_tests.sh --skeleton
+        run: |
+          # main test suite
+          cmake --build . --target check
```
Comment (sergeyklay) [inline .github/workflows/ci.yml:267]: It should be:

```suggestion
          cmake --build ${{ matrix.build-dir }} --target check
```

Comment (skvadrik) [reply .github/workflows/ci.yml:267]: But I have added `working-directory: ${{ matrix.build-dir }}`, doesn't it mean that `.` is `${{ matrix.build-dir }}` ?

Comment (sergeyklay) [reply .github/workflows/ci.yml:267]: Oh! You're right!

Comment (skvadrik) [reply .github/workflows/ci.yml:267]: Also, the tests do run.

**Pull request merged**